chat-plugins/info.js

```diff
@@ -1821,10 +1822,10 @@ var commands = exports.commands = {
 			}
 			return this.sendReplyBox("Random number " + num + "x(1 - " + faces + "): " + rolls.join(", ") + "<br />Total: " + total);
 		}
-		if (target && isNaN(target) || target.length > 21) return this.sendReply("The max roll must be a number under 21 digits.");
```

Comment:
You are right... Yea, I guess we can keep it, but call <code>Math.floor</code> rather than <code>parseInt</code> -since they are supposed to be positive numbers.

I can't believe there is no direct way to parse only base 10 numerals while supporting scientific notation :| I mean, scientific notation uses base 10 <_>

The biggest issue with <code>parse[Int/Float]</code> is that they won't return <code>NaN</code> if the string starts with a number and ends with gibberish... This can potentially lead to confusion as to the meaning of <code>d</code> and other symbols, but it's also not a big deal.